src/CLI/actioner/cli_client.py

```diff
@@ -23,9 +23,15 @@
 import pwd
 import os
 from six.moves.urllib.parse import quote
+import syslog
+import requests
 
 urllib3.disable_warnings()
 
+sess = requests.Session()
```

Comment:
Since the sess is scoped within the ApiClient class, can you define it as a "class" variable for AplClient instances to share the persistent session?

Reply:
This forces the instance of ApiClient to be global. It is not global in majority of the python actioner scripts.